docs/considerations.rst

```diff
@@ -0,0 +1,183 @@
+##########################
+Considerations for ME-fMRI
+##########################
+Multi-echo fMRI acquisition sequences and analysis methods are rapidly maturing. Someone who has access
+to a multi-echo fMRI sequence should seriously consider using it. The following are a few points to 
+consider when deciding whether or not to collect multi-echo data
+
+Possible increase in TR
+-----------------------
+The one difference with multi-echo is a slight time cost.
```

Comment:
What about heating?

Reply:
I'll add some bits about possible gradient heating - maybe in the FAQ (and maybe also separated out the FAQ into multiecho specific (ie - does ME-fMRI increase heating for subjects?) and tedana specific (what is there right now).

Reply:
I'd say gradient heating is an acquisition note rather than a "cost of ME", personallyl ! Maybe that's a judgement call